dev-cli/container/src/node/src/defaults.js

```diff
@@ -12,5 +14,7 @@ export const AoModuleTags = [
   DEFAULT_INPUT_ENCODING_TAG,
   DEFAULT_OUTPUT_ENCODING_TAG,
   DEFAULT_VARIANT_TAG,
+  DEFAULT_MEMORY_LIMIT_TAG,
+  DEFAULT_COMPUTE_LIMIT_TAG,
```

Comment:
This shouldn't be in this PR. There needs to be internal discussion on if there should even _be_ defaults for this or if it should be derived from the build flags used to produce the wasm binary (see #983)

Let's keep this PR small and focused on only the issues highlighted in the issues it claims to address.